scripts/pegasus_rolling_update.sh

```diff
@@ -123,6 +123,14 @@ if [ $set_ok -ne 1 ]; then
   exit 1
 fi
 
+echo "Set lb.assign_delay_ms to 60min..."
```

Comment:
I think 60min is too long, I review the log yesterday, it cost 11 minutes to initialization. How about update to 30min?

Reply:
OK